### PR TITLE
Strip DEM index changes

### PR DESCRIPTION
**Pull request by clairecporter**
- Change strip dem index fields to use Boolean and DatTeime where appropriate
- Make a release-style option for strip DEMs to match the tile changes made in [PR#74](https://github.com/PolarGeospatialCenter/pgcdemtools/pull/74)
- Update packaging scripts to use release-style field names
- Deprecate registration source fields

### CODE REVIEW

**Comment by clairecporter** [comment]
Once we are ready to merge this in, we'll have a pause on DEM ingest while @dannyim and @ehusby and I update the repos and modify the sandwich tables to use the new datatypes.

File: index_setsm.py [hunk 1]
```diff
@@ -534,7 +546,13 @@ def write_to_ogr_dataset(ogr_driver_str, ogrDriver, dst_ds, dst_lyr, groups, pai
         if not layer:
             logger.info("Creating table...")
 
-            layer = ds.CreateLayer(dst_lyr, tgt_srs, ogr.wkbMultiPolygon)
+            # FileGDB will throw a warning when inserting datetimes without this
+            if ogr_driver_str in ['FileGDB', 'OpenFileGDB']:
+                co = ['TIME_IN_UTC=NO']
```
Comment (ehusby) [inline index_setsm.py:551]: Cool!

**Review by ehusby — APPROVED** [review]
Generally looks good to me! As long as all is working for both tile and strip index creation, we're good.

**Comment by clairecporter** [comment]
I'm testing on tile indexing and packaging now.

**Pull request merged**